src/ILVerification/src/TypeVerifier.cs

```diff
@@ -122,26 +123,69 @@ private string Format(DefType defType, InterfaceImplementationHandle interfaceIm
         {
             if (_verifierOptions.IncludeMetadataTokensInErrorMessages)
             {
-                return $"{defType}(0x{_module.MetadataReader.GetToken(interfaceImplementationHandle)})";
+
+                return string.Format("{0}([{1}]0x{2:X8})", defType, GetModule(defType), _module.MetadataReader.GetToken(interfaceImplementationHandle));
             }
             else
             {
                 return defType.ToString();
             }
         }
 
+        private string GetModule(DefType defType)
+        {
+            InstantiatedType instantiatedType = defType as InstantiatedType;
```

Comment:
It is better to write it this way:

```
if (defType is InstantiatedType instantiatedType)
{
...
}
```

Reply:
I was thinking about it when I was taking a shower :smile:

Reply:
fixed thank's!